react-native/react/tabs/devices/devices-render.desktop.js

```diff
@@ -9,32 +9,69 @@ import HardwareComputer from 'material-ui/lib/svg-icons/hardware/computer'
 import CommunicationVpnKey from 'material-ui/lib/svg-icons/communication/vpn-key'
 import ActionNoteAdd from 'material-ui/lib/svg-icons/action/note-add'
 
+import moment from 'moment'
 import View from 'react-flexbox'
 
-export default class Devices extends BaseComponent {
-
+export default class DevicesRender extends BaseComponent {
   constructor (props) {
     super(props)
   }
 
-  connectNew () {
-    console.log("Add device")
+  renderPhone (device) {
+    return (
+      <div key={device.deviceID} style={Object.assign({}, styles.deviceOuter, styles.deviceShow)}>
```

Comment:
??? you can do 
{...styles.deviceOuter, styles.deviceShow} and it'll make the same object as object.assign